index.js

```diff
@@ -59,20 +59,26 @@ function ShopifyToken(options) {
   this.apiKey = options.apiKey;
 }
 
+ShopifyToken.prototype.generateNonce = function() {
```

Comment:
Please add JSDoc for this method and also add documentation for it in the README.